spec/order_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'order'
+require 'stringio'
+
+describe Order do
+  let(:menu) { double :menu,
```

Comment:
Style/Blocks: Avoid using {...} for multi-line blocks.<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.